src/components/CTable.vue

```diff
@@ -62,30 +90,52 @@ export default {
       if (!this.filter) {
         return this.messages;
       }
-      const filterKeys = ['name', 'message_id'];
+      const filterKeys = ['name', 'messageId', 'actorName'];
+
       return this.messages.filter(m =>
         filterKeys
           .map(key => m[key].toLowerCase())
           .join('~~')
           .includes(this.filter.toLowerCase())
       );
+    },
+    displayedMessages() {
+      if (!this.sortedColumn) {
+        return this.filteredMessages;
+      }
+      return [...this.filteredMessages].sort((row1, row2) => {
+        const direction = this.sortDirection === 'asc' ? 1 : -1;
+        const a = row1[this.sortedColumn] || 0,
+          b = row2[this.sortedColumn] || 0;
+        return direction * (a - b);
+      });
+    }
+  },
+  methods: {
+    setSortColumnAndDirection(columnName) {
+      if (columnName === this.sortedColumn) {
```

Comment:
Normally when you click a third time on a column it deactivate the sort.

You should add a condition if  columnName === this.sortedColumn and sort direction  === 'desc' it should also set sortedColumn to null.